XML3-Homework/erin_HW3.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<root>
+    <title meaning="My Moon">"Ma Lune"</title>
+    <metadata lang="French" band="Zaho"/>
+    <video link="https://www.youtube.com/watch?v=EjDnBXKaehY"/>
+
+    <p>
+        <n> Je t'ai cherché, je visais <moon>la lune</moon></n>
+        <n> Tu m'as connue le cœur dans la brume</n>
+        <n> Quand l'amour a déserté mon île</n>
+        <n> J'ai écrit entre le marteau et l'enclume</n>
+        <n> Je ne suis pas née dans <riches>la fortune</riches></n>
+        <n> Ma seule <riches>richesse</riches> : ma famille, mes coutumes</n>
+        <n> Je fais l'or dans mes disques et ma plume</n>
+        <n> Et tu seras éternel dans mes projets posthumes</n>
+        <n> Les regards ne seront pas unanimes</n>
+        <n> Pourtant, fermer les yeux est facile</n>
+        <n> J'ai fait semblant, j'ai fait face au vent</n>
+        <n> Mais face au miroir, je ne vois qu'un enfant</n>
+        <n> Vaut-il mieux être anonyme ?</n>
+        <n> Je n'sais plus qui m'aime ou qui m'abîme</n>
+        <n> Que restera-t-il avec le temps ?</n>
+        <n> Avec le temps, yeah, yeah</n>
+    </p>
+    <br/>
```

Comment:
@erinmooney You don't really need this `<br/>` element, do you? The `<p>` element makes a nice block to group the `<n>` elements inside.